src/components/freelancer/interviewProfile/interviewProfile.tsx

```
@@ -475,37 +474,43 @@ const InterviewProfile: React.FC<{ freelancerId: string }> = ({
-                            <Edit2 className="w-4 h-4 text-gray-400 cursor-not-allowed" />
-                          }
-                          disabled
-                          onClick={() =>
-                            handleSkillDomainDialog(
-                              item,
-                              item.experience ? 'SKILL' : 'DOMAIN',
-                            )
-                          }
-                        />
+                      <TableCell className="py-2">
+                        {String(item?.interviewStatus).toUpperCase() === 'COMPLETED' ? (
```

ESLint

[error] Insert `⏎·······················`

(prettier/prettier)